.github/workflows/tests.yml

```diff
@@ -85,10 +85,10 @@ jobs:
           set -ex
           source tools/github/before_install.sh
           set -ex
-          python setup.py sdist
           if [[ $INSTALL_FROM_SDIST == "1" ]]; then
-              pip uninstall cython -y;
-              pip install dist/scikit-image-*.tar.gz;
+              pip uninstall cython -y
+              python setup.py sdist
+              pip install dist/scikit-image-*.tar.gz
           else
               pip install -vv -e .;
```

Comment:
So is our CI actually testing the new workflow and build system in `meson.md`? As mentioned in https://github.com/scikit-image/scikit-image/pull/6536#discussion_r995193616, this is actually using setuptools under the hood. And the `setup.py sdist` approach doesn't use meson either.

Are you waiting with CI integration until `dev.py` is added?

Reply:
No, if you look at the build logs, you'll see meson is used in various places, since the build system in pyproject.toml has been changed to Meson.

Reply:
Yes, I can see that meson is used. However, as mentioned in the linked comment [`pip install -e .` (in-place install)](https://github.com/scikit-image/scikit-image/actions/runs/3292132986/jobs/5427133838#step:4:146) uses setuptools and when I tested this locally there was no parallel build happening for this command. Just the slow consecutive build with setuptools. I'm asking whether we should also test the explicit workflow that doesn't use a in-place install, e.g. invoking `ninja`. And take advantage of the parallel feature in our CI. 

Maybe I'm just misunderstanding something; this interaction between the different tools and build systems still seems very opaque to me.

Reply:
Discussed in the community meeting: the CI test's a non-editable install from an sdist. Also it seems that `pip install -e .` does both, building with ninja *and* building with setuptools. So either way the CI is testing the meson build.